docs/src/refs.bib

```diff
@@ -1,4 +1,158 @@
 
+@Article{BS84,
+  author        = {Beynon, W. M. and Spaltenstein, N.},
+  title         = {Green functions of finite {Chevalley} groups of type $E_n (n=6,7,8)$},
+  zbl           = {0539.20025},
+  journal       = {J. Algebra},
+  fjournal      = {Journal of Algebra},
+  volume        = {88},
+  pages         = {584--614},
+  year          = {1984},
+  doi           = {10.1016/0021-8693(84)90084-X},
+  language      = {English},
+  zbmath        = {3857372}
+}
+
+@InCollection{CR74,
+  author        = {Chang, Bomshik and Ree, Rimhak},
+  title         = {The characters of $G_2(q)$},
+  booktitle     = {Symposia {M}athematica, {V}ol. {XIII} ({C}onvegno di {G}ruppi {A}beliani \& {C}onvegno di {G}ruppi e
+                  loro {R}appresentazioni, {INDAM}, {R}ome, 1972)},
+  mrnumber      = {364419},
+  publisher     = {Academic Press, London-New York},
+  pages         = {395--413},
+  year          = {1974}
+}
+
+@Misc{Car85,
+  author        = {Carter, Roger W.},
+  title         = {Finite groups of {Lie} type. {Conjugacy} classes and complex characters},
+  zbl           = {0567.20023},
+  howpublished  = {Pure and {Applied} {Mathematics}. {A} {Wiley}-{Interscience} {Publication}. {Chichester}-{New} {York}
+                  etc.: {John} {Wiley} and {Sons}. {XII}, 544 p. £42.50 (1985).},
+  year          = {1985},
+  language      = {English},
+  zbmath        = {3904833}
```

Comment:
Better:
```
@book {MR794307,
    AUTHOR = {Carter, Roger W.},
     TITLE = {Finite groups of {L}ie type},
    SERIES = {Pure and Applied Mathematics (New York)},
      NOTE = {Conjugacy classes and complex characters,
              A Wiley-Interscience Publication},
 PUBLISHER = {John Wiley \& Sons, Inc., New York},
      YEAR = {1985},
     PAGES = {xii+544},
      ISBN = {0-471-90554-2},
   MRCLASS = {20G40 (20-02 20C15)},
  MRNUMBER = {794307},
MRREVIEWER = {David\ B.\ Surowski},
}
```
The first line of the "note" probably should be a `subtitle` (not sure if `DocumenterCitation` supports those)

Reply:
It seems to be fine with `subtitle`.